src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -561,6 +568,12 @@ export class SearchView extends ViewPane {
 			const collapsed = nodeExists ? undefined :
 				(collapseResults === 'alwaysCollapse' || (fileMatch.matches().length > 10 && collapseResults !== 'alwaysExpand'));
 
+			// {{ SQL CARBON EDIT }}
+			// if serach is launced from books viewlet, do not show collapsible tree
```

Comment:
Don't need to mention books here since is this just a general property anyone calling this could set